wikimedia/wikipedia-ios
# Image Recommendations - Fetch images in WKData

**Phabricator:** https://phabricator.wikimedia.org/T358933

### Notes
This PR adds the ability to fetch images from WKData, and integrates it into Image Recommendations.

### Test Steps
1. Go to image recommendations on Staging
2. Confirm image is populated in half sheet modal
3. Tap Yes. Confirm image is populated in add image details view.
4. Dismiss. Tap Not Sure. Confirm image refreshes with next recommendation.
